### DISCUSSION TITLE
Grey-out completed elements in todo lists

### DISCUSSION THREAD

**Discussion by octvs** [jakewvincent/mkdnflow.nvim#207] [Q&A] [answered]
Hey there, 
It might fall out of the scope of mkdnflow, but is it possible to grey-out the
completed to-dos in a list, based on the status of their checkboxes? 
Cheers

**Comment by jakewvincent** [comment, marked as answer]
Hi @octvs, just giving you a heads up that I am taking this up. Thanks for the idea!

**Comment by octvs** [comment]
Hi there as far as I see from the latest updates it is now possible to achieve
this easily with

```lua
to_do = {
   highlight = true,
}
```

As a follow-up; I fail to override the default highlight color for status
'not_started'. I would prefer to have it not concealed, simply setting that
singular option throws me an error.

```lua
to_do = { 
    highlight = true,
    statuses = {
        {
            name = "not_started",
            highlight = { content = { link = "String" }, },
        }
    }
}
```

Same went along when I tried the copy-over the complete table.
